packages/nitro-protocol/scripts/postdeploy.js

```diff
@@ -0,0 +1,14 @@
+const {writeFileSync} = require('fs');
+
+const jsonPath = __dirname + '/../addresses.json';
+const addresses = require(jsonPath);
+
+function deepDelete(object, keyToDelete) {
+  Object.keys(object).forEach(key => {
+    if (key === keyToDelete) delete object[key];
+    else if (typeof object[key] === 'object') deepDelete(object[key], keyToDelete);
+  });
+}
+const keyToDelete = 'abi';
+deepDelete(addresses, keyToDelete);
```

Comment:
Conceptually, it makes more sense to me to _keep the keys you want_, rather than _delete they keys you don't want_.

Is there a reason for the latter here?

Reply:
Reason 1: this is just transcribing existing code, so I considered it safer to adopt the existing approach. 
Reason 2: it's a deeply nested object, so there are fare more keys to keep than to delete in this case. Although I afree with your instinct in general, I think in this case it would be more complicated to implement it that way. It's marginal.